src/constants/links.js

```diff
@@ -1,14 +1,15 @@
 export const header = {
-  navDropDowns: [ // dropdowns
+  navDropDowns: [
+    // dropdowns
     // first dropdown start
     [
       {
         LABEL: 'navigation.dropdowns.0.linkLabels.0',
-        LINK: 'link-one',
+        LINK: 'home',
       },
       {
         LABEL: 'navigation.dropdowns.0.linkLabels.1',
-        LINK: 'link-two',
+        LINK: 'documentation',
```

Comment:
`/documentation`?

Reply:
Doesn't route correctly when the slash is included

Reply:
It must have to do with the logic in our custom link component?